platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/constants/MapboxConstants.java

```diff
@@ -117,6 +117,16 @@
    */
   public static final String FRAG_ARG_MAPBOXMAPOPTIONS = "MapboxMapOptions";
 
+  /**
+   * CN base url API endpoint
+   */
+  public static final String URL_API_CN = "api.mapbox.cn";
+
+  /**
+   * CN base url events endpoint
+   */
+  public static final String URL_EVENTS_CN = "https://events.mapbox.cn";
```

Comment:
it doesn't look like we actually need to maintain url to china endpoint here, it should be configurable via factory class: https://github.com/mapbox/mapbox-events-android/blob/4c40e3d4cd0fc5ca882d15b16120776d17c9dd0b/libtelemetry/src/main/java/com/mapbox/android/telemetry/MapboxTelemetry.java#L313